apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/StandardGrpcRdaSourceTest.java

```diff
@@ -86,8 +87,8 @@ public class StandardGrpcRdaSourceTest {
   /** String used as a RDA API "version" in the unit tests. */
   public static final String VERSION = "version";
 
-  /** A MetricRegistry used to verify metrics. */
-  private MetricRegistry appMetrics;
+  /** A MeterRegistry used to verify metrics. */
```

Comment:
minor nit; we've been trying to use things like {@link MetricRegistry} in JavaDoc(s)